config/config.go

```diff
@@ -23,14 +26,17 @@ func Get() (*Config, error) {
 		return cfg, nil
 	}
 
+	defaultTimeout := time.Duration(5 * time.Second)
+
 	cfg = &Config{
 		BindAddr:                ":22100",
 		Brokers:                 []string{"localhost:9092"},
 		FilterJobSubmittedTopic: "filter-job-submitted",
-		Host:                    "http://localhost:22100",
-		KafkaMaxBytes:           "2000000",
-		PostgresURL:             "user=dp dbname=FilterJobs sslmode=disable",
-		SecretKey:               "FD0108EA-825D-411C-9B1D-41EF7727F465",
+		Host:            "http://localhost:22100",
+		KafkaMaxBytes:   "2000000",
+		PostgresURL:     "user=dp dbname=FilterJobs sslmode=disable",
+		SecretKey:       "FD0108EA-825D-411C-9B1D-41EF7727F465",
+		ShutdownTimeout: defaultTimeout,
```

Comment:
Not sure why your assigning a variable for this. Can you just inline `ShutdownTimeout: 5 * time.Second`